sdks/python/apache_beam/runners/common.py

```diff
@@ -1262,7 +1262,7 @@ def finalize(self):
 
   def _reraise_augmented(self, exn):
     if getattr(exn, '_tagged_with_step', False) or not self.step_name:
-      raise
+      raise exn
```

Comment:
Just explicitly reraised the active exception.

Reply:
lgtm